dbms/src/Common/MyDuration.cpp

```diff
@@ -67,4 +67,10 @@ String MyDuration::toString() const
     auto frac_str = fmt::format("{:06}", microsecond);
     return fmt::format(fmt_str, sign > 0 ? "" : "-", hour, minute, second, frac_str);
 }
-} // namespace DB
\ No newline at end of file
+
+UInt64 calcSeconds(int hour, int minute, int second)
```

Comment:
I think it's better to use `Int32` as the argument types. It should be the same as `MyDuration::hours()/minutes()/second()`'s return type.